### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by JohnQuaresma**

### CODE REVIEW

**Comment by jrobeson** [comment]
ping?

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 ember-cli
```
Comment (rwjblue) [inline LICENSE.md]: Can you change this to "Ember CLI Contributors"?

Comment (stefanpenner) [inline LICENSE.md]: @JohnQuaresma ^^

**Comment by rwjblue** [comment]
Added manually.

**Pull request closed without merging**